src/views/faq/l10n.json

```diff
@@ -86,7 +86,7 @@
     "faq.sellProjectsTitle":"Can I sell my Scratch projects?",
     "faq.sellProjectsBody":"Certainly - your project is your creation. Keep in mind that once you share your project on Scratch, everyone is free to download, remix, and reuse it as per the terms of the <a href=\"http://creativecommons.org/licenses/by-sa/2.0/deed.en\">CC-BY-SA 2.0 license</a>. So if you intend to sell your project, you may want to un-share it from Scratch.",
     "faq.sourceCodeTitle":"Where can I find the source code for Scratch?",
-    "faq.sourceCodeBody":"The source code for Scratch 2 (website and editor) is not yet available. The source code for <a href =\"/scratch_1.4\">Scratch 1.4</a>, written in Squeak, is available <a href=\"https://github.com/LLK/Scratch_1.4\">on GitHub</a>. The source code for the first version of the Scratch website - ScratchR - is available <a href=\"http://svn.assembla.com/svn/scratchr/\">on Assembla</a>.",
+    "faq.sourceCodeBody":"The source code for the Scratch 2 editor can be found <a href=\"https://github.com/LLK/scratch-flash\">on GitHub</a>. The source code for <a href =\"/scratch_1.4\">Scratch 1.4</a>, written in Squeak, is also available <a href=\"https://github.com/LLK/Scratch_1.4\">on GitHub</a>. For updated information on development projects relating to the Scratch website, please visit our <a href=\"/developers\">Developer Page</a>.",
```

Comment:
small tweak if you have a moment - in both of the Github links, could we take the word "on" out of the anchor and have it appear beforehand?